clinical/dbaccess.py

```diff
@@ -13,112 +13,6 @@
 import psutil
 import subprocess
 
-def readconfig( config ):
-  """Reads parameters from a config file.
-
-  Args:
-    config (str): path to config file
-    If config does not exist the default will be used
-  
-  Returns:
-    dict: parameters from the config file (unparsed)
-  """
-  if os.path.isfile(config):
-    Configfile = config
-  else:
-    Configfile = "/home/hiseq.clinical/.scilifelabrc"
-  Params = {}
-  with open(Configfile, "r") as Confs:
-    for line in Confs:
-      if len(line) > 5 and not line[0] == "#":
-        line = line.rstrip()
-        pv = line.split(" ")
```

Comment:
Not completely obvious what ``pv`` is a reference to here. Since it's often possible to avoid excessive commenting by just naming variables more explicitly I think this would be a good improvement here!